cription: pom.xml defines a dependency on trustmanager 3.1.4
 The assembly will embed trustmaanger 3.1.4 in the lib directory.
Author: Valery Tschopp <valery.tschopp@switch.ch>
--- a/pom.xml
+++ b/pom.xml
@@ -29,6 +29,12 @@
     </repositories>
 
     <dependencies>
+        <!-- Debian6 patch: uses trustmanager 3.1.4 -->
+        <dependency>
+            <groupId>emi</groupId>
+            <artifactId>trustmanager</artifactId>
+            <version>3.1.4</version>
+        </dependency>
         <!-- Debian6 patch: OS provided bouncycastle is too old -->
         <dependency>
             <groupId>org.bouncycastle</groupId>
